Sources/ArcGISToolkit/Components/Offline/MetadataView.swift

```diff
@@ -0,0 +1,128 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct MetadataView: View {
+    /// The view model for the preplanned map.
+    @ObservedObject var model: PreplannedMapModel
+    
+    /// The action to dismiss the view.
+    @Environment(\.dismiss) private var dismiss: DismissAction
+    
+    public var body: some View {
+        Form {
+            Section {
+                VStack(alignment: .leading) {
+                    if let image = model.preplannedMapArea.thumbnail {
+                        VStack(alignment: .leading, spacing: 0) {
+                            Text("Thumbnail")
+                                .font(.caption)
+                                .foregroundStyle(.secondary)
+                            LoadableImageView(loadableImage: image)
+                                .clipShape(.rect(cornerRadius: 10))
+                                .padding(.vertical, 10)
+                        }
+                        Divider()
+                    }
+                    VStack(alignment: .leading, spacing: 0) {
+                        Text("Name")
+                            .font(.caption)
+                            .foregroundStyle(.secondary)
+                        Text(model.preplannedMapArea.title)
+                            .font(.subheadline)
+                    }
+                    Divider()
+                    VStack(alignment: .leading, spacing: 0) {
+                        Text("Description")
+                            .font(.caption)
+                            .foregroundStyle(.secondary)
+                        Text(model.preplannedMapArea.description)
+                            .font(.subheadline)
```

Comment:
An `isEmpty` check should be added for the description:

```suggestion
                        if !model.preplannedMapArea.description.isEmpty {
                            Text(model.preplannedMapArea.description)
                                .font(.subheadline)
                        } else {
                            Text("This area has no description.")
                                .font(.subheadline)
                                .foregroundStyle(.tertiary)
                        }
```
<img src="https://github.com/user-attachments/assets/d5e998a6-a344-4831-8c6a-675b40491f8a" width="300">